src/components/notification/Notification.js

```diff
@@ -4,7 +4,13 @@ import { defineMessages, injectIntl } from 'react-intl';
 import classNames from 'classnames';
 
 // $FlowFixMe
-import { Information, Check, AlertTriangle, AlertBadge, XMark } from '@box/blueprint-web-assets/icons/Medium';
+import {
+    InformationCircle,
+    CheckmarkCircle,
+    AlertTriangle,
+    XMark,
+    XMarkCircle,
+} from '@box/blueprint-web-assets/icons/Medium';
```

Comment:
apparently upon upgrading` blueprint-web-assets`, there were deprecations and renames so it broke these tests